docs/guide/using-doctrine-orm-filters.php

```diff
@@ -186,24 +186,6 @@ function configure(ContainerConfigurator $configurator) {
      }
  }
 
-namespace App\Configurator {
```

Comment:
Same: why removing this configuration?

Reply:
The convention for configuration in guides is to namespace by `App\DependencyInjection` : cf [pdg application Kernel](https://github.com/JacquesDurand/php-documentation-generator/blob/main/tests/Application/src/Kernel.php#L96)